_utility/deploy_prb.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+if [ "$#" -ne 3 ]; then
+  echo "Invalid argument count."
```

Comment:
I would say something like this: 
"Insufficient arguments, you have only provided: "$@", see usage and example for further assistance.".